src/OpenApiRuntime/Client/Psr7HttplugEndpointTrait.php

```diff
@@ -10,12 +10,17 @@
 
 trait Psr7HttplugEndpointTrait
 {
-    abstract protected function transformResponseBody(string $body, int $status, SerializerInterface $serializer);
+    abstract protected function transformResponseBody(string $body, int $status, SerializerInterface $serializer, ?string $contentType = null);
```

Comment:
I think the default `$contentType` should be empty string, not null.

And the `?` can be removed.